tests/integration/test_plugins_dedup_batch4.py

```
@@ -0,0 +1,1653 @@
+    def test_list_files_happy_path(self) -> None:
+        from file_organizer.plugins.sdk.client import PluginClient
+
+        items = [{"name": "file.txt", "path": "/data/file.txt"}]
+        response_payload = {"items": items}
+
+        def handler(request: httpx.Request) -> httpx.Response:
+            return httpx.Response(200, json=response_payload)
+
+        client = PluginClient(
+            base_url="http://localhost:8000",
+            token="tok",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "token"

(S106)